ui/ui-components/pages/index.tsx

```diff
@@ -8,7 +8,7 @@ export default function Page(props) {
 
   let CTALink = "/session/sign-in";
   let CTAMessage = "Sign In";
-  let CTATarget: string;
+  let CTATarget = null;
```

Comment:
You may not have to lose the type definition here. Could you do something like this:

```ts
let CTATarget?: string;
```

Or this:

```ts
let CTATarget: string | null = null;
```

Not sure exactly what you're protecting against with this change.